Android/app/src/main/res/layout/fragment_my_trips.xml

```diff
@@ -15,7 +15,10 @@
         app:lottie_fileName="loading.json"
         app:lottie_imageAssetsFolder="images"
         app:lottie_loop="true" />
-
+<android.support.v4.widget.SwipeRefreshLayout
```

Comment:
The indentation is not proper here. Checkstyle will fail.

Reply:
@Swati4star changes done!